src/components/Game.tsx

```diff
@@ -0,0 +1,13 @@
+import { FC } from 'react';
+import { Board } from './Board';
+
+const ROWS = 4;
+const COLS = 4;
+
+export const Game: FC = (): JSX.Element => {
+  return (
+    <div>
+      <Board cols={COLS} rows={ROWS} />
+    </div>
```

Comment:
Macke two component:
- Background (or real board).
- Cells(or something like this) that will represent cells.

Reply:
Should i make it inside Board component or instead of Board component?